### ISSUE TITLE
Cannot find module 'commander'

### ISSUE THREAD

**Issue by liudonghua123** [chenquincy/app-info-parser#62]
**Describe the bug**
Cannot find module 'commander'

**To Reproduce**
1. npx app-info-parser -f some-app.apk
2. or `npm i -g app-info-parser && app-info-parser -f some-app.apk`

**Expected behavior**
Work expected.

```shell
C:\Users\Liu.D.H\Documents\WeChat Files\iman405904\FileStorage\File\2021-06>npx app-info-parser -f com.cdvcloud.qicaiyun_305.apk
npm notice
npm notice New minor version of npm available! 7.10.0 -> 7.16.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v7.16.0
npm notice Run npm install -g npm@7.16.0 to update!
npm notice
Need to install the following packages:
  app-info-parser
Ok to proceed? (y)
node:internal/modules/cjs/loader:943
  throw err;
  ^

Error: Cannot find module 'commander'
Require stack:
- C:\Users\Liu.D.H\AppData\Local\npm-cache\_npx\5d235a1832a4f7be\node_modules\app-info-parser\bin\index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:940:15)
    at Function.Module._load (node:internal/modules/cjs/loader:773:27)
    at Module.require (node:internal/modules/cjs/loader:1012:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at Object.<anonymous> (C:\Users\Liu.D.H\AppData\Local\npm-cache\_npx\5d235a1832a4f7be\node_modules\app-info-parser\bin\index.js:3:21)
    at Module._compile (node:internal/modules/cjs/loader:1108:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
    at Module.load (node:internal/modules/cjs/loader:988:32)
    at Function.Module._load (node:internal/modules/cjs/loader:828:14)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\Liu.D.H\\AppData\\Local\\npm-cache\\_npx\\5d235a1832a4f7be\\node_modules\\app-info-parser\\bin\\index.js'
  ]
}

C:\Users\Liu.D.H\Documents\WeChat Files\iman405904\FileStorage\File\2021-06>npm install app-info-parser -g

added 22 packages, and audited 23 packages in 6s

3 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

C:\Users\Liu.D.H\Documents\WeChat Files\iman405904\FileStorage\File\2021-06> app-info-parser -f com.cdvcloud.qicaiyun_305.apk
node:internal/modules/cjs/loader:943
  throw err;
  ^

Error: Cannot find module 'commander'
Require stack:
- C:\Users\Liu.D.H\AppData\Roaming\npm\node_modules\app-info-parser\bin\index.js
[90m    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:940:15)[39m
[90m    at Function.Module._load (node:internal/modules/cjs/loader:773:27)[39m
[90m    at Module.require (node:internal/modules/cjs/loader:1012:19)[39m
[90m    at require (node:internal/modules/cjs/helpers:93:18)[39m
    at Object.<anonymous> (C:\Users\Liu.D.H\AppData\Roaming\npm\node_modules\[4mapp-info-parser[24m\bin\index.js:3:21)
[90m    at Module._compile (node:internal/modules/cjs/loader:1108:14)[39m
[90m    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)[39m
[90m    at Module.load (node:internal/modules/cjs/loader:988:32)[39m
[90m    at Function.Module._load (node:internal/modules/cjs/loader:828:14)[39m
[90m    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)[39m {
  code: [32m'MODULE_NOT_FOUND'[39m,
  requireStack: [
    [32m'C:\\Users\\Liu.D.H\\AppData\\Roaming\\npm\\node_modules\\app-info-parser\\bin\\index.js'[39m
  ]
}

C:\Users\Liu.D.H\Documents\WeChat Files\iman405904\FileStorage\File\2021-06>
```

I found 'commander' dependence is a dev dependence, maybe this is the problem.

**Comment by liudonghua123** [comment]
If I go to `C:\Users\Liu.D.H\AppData\Roaming\npm\node_modules\app-info-parser` and execute `npm i`, then it was ok.

**Comment by chenquincy** [comment]
@liudonghua123 Thx for your issue. It have been fixed in v1.1.1
